samples/net/openthread/ncp/prj.conf

```diff
@@ -0,0 +1,109 @@
+# Generic networking options
+CONFIG_NETWORKING=y
+CONFIG_NET_UDP=y
+CONFIG_NET_TCP=n
+CONFIG_NET_IPV6=y
+CONFIG_NET_IPV4=n
+
+CONFIG_NET_CONFIG_NEED_IPV6=y
+CONFIG_NET_CONFIG_NEED_IPV4=n
+
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_NET_SOCKETS_POLL_MAX=4
+#CONFIG_NET_CONNECTION_MANAGER=y
+
+# disable certain parts of Zephyr IPv6 stack
+CONFIG_NET_IPV6_NBR_CACHE=n
+CONFIG_NET_IPV6_MLD=n
+
+# Kernel options
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_INIT_STACKS=y
+#CONFIG_DEBUG_OPTIMIZATIONS=y
+
+# Logging
+CONFIG_NET_LOG=y
+CONFIG_LOG=y
+CONFIG_NET_STATISTICS=y
+CONFIG_PRINTK=y
+
+# Network buffers
+CONFIG_NET_PKT_RX_COUNT=16
+CONFIG_NET_PKT_TX_COUNT=16
+CONFIG_NET_BUF_RX_COUNT=80
+CONFIG_NET_BUF_TX_COUNT=80
+CONFIG_NET_CONTEXT_NET_PKT_POOL=y
+
+# Thread by default registers quite a lot addresses.
+CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=6
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=8
+CONFIG_NET_MAX_CONTEXTS=10
+
+# Network shell
+# CONFIG_NET_SHELL=y
+
+# ???
```

Comment:
I think we don't need `net_config` or shell.

Reply:
removed.